pytest/tests/delete_remote_nodes.py

```diff
@@ -11,7 +11,7 @@
 from utils import user_name
 
 machines = gcloud.list()
-to_delete_prefix = sys.argv[1] if len(sys.argv) >= 2 else f"pytest-node-{user_name()}"
+to_delete_prefix = sys.argv[1] if len(sys.argv) >= 2 else f"pytest-node-{user_name()}-"
```

Comment:
is this change intended?

Reply:
yes. when i trying delete bo-node, bowen-node was selected too :joy:

Reply:
unfortunate :)